hadoop-ozone/dist/src/main/license/bin/LICENSE.txt

```diff
@@ -228,12 +228,12 @@ BSD
 CDDL
 =====================
 
+   com.sun.istack:istack-commons-runtime
    com.sun.jersey:jersey-core
    com.sun.jersey:jersey-json
    com.sun.jersey:jersey-server
    com.sun.jersey:jersey-servlet
    com.sun.xml.bind:jaxb-core
```

Comment:
I think this one can be removed, too.

Reply:
Yep. We already have the line for the new jaxb-core: `org.glassfish.jaxb:jaxb-core`